src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java

```diff
@@ -72,9 +77,16 @@ public void anonCanRemindPasswordOnUserLoginSpecialPageUsingLowerCaseUserName()
     String
         newPassword =
         login.receiveMailWithNewPassword(credentials.email, credentials.emailPassword);
-    signIn.login(userName, newPassword);
     String verifyString = userName.substring(0, 1).toUpperCase() + userName.substring(1);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "verifyString" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)